atomic_reactor/plugins/pre_add_help.py

```diff
@@ -84,4 +91,4 @@ def run(self):
 
         self.log.info("added %s", man_path)
 
-        return content
+        return self.HELP_GENERATED
```

Comment:
Yes, its possible although I wanted to avoid two things here:
 - return `None` if exception and an empty string if no help file was used
 - duplicating information from the config